ProcessMaker/Http/Controllers/Api/Requests/RequestsController.php

```diff
@@ -40,4 +44,70 @@ public function index(Request $request)
         // Return fractal representation of paged data
         return fractal($requests, new ApplicationTransformer())->respond();
     }
+
+    /**
+     * Returns the list of processes that can be started by the connected user
```

Comment:
Where is it verified that users can create Requests?

Reply:
Here we have the problem that we aren't managing assignment rules yet. So for the moment we don't have the documentation. @velkymx  I suggest to create a new ticket to implement this.